### PR TITLE
Disallow unusable sizes for farming

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: src/plotting/create_plots.py [hunk 1]
```diff
@@ -82,9 +82,9 @@ def create_plots(args, root_path, use_datetime=True, test_private_keys: Optional
 
     if args.size < config["min_mainnet_k_size"] and test_private_keys is None:
         log.warning(f"Creating plots with size k={args.size}, which is less than the minimum required for mainnet")
-    if args.size < 22:
-        log.warning("k under 22 is not supported. Increasing k to 22")
-        args.size = 22
+    if args.size < 25:
```
Comment (jespino) [inline src/plotting/create_plots.py:85]: I'm not sure if we want to do something with the sizes between 25 and 32 here. I changed it in the `click` interface, but I'm not sure about the need of changing it internally. Actually I'm not 100% sure about the need of changing it here from 22 to 25.

@hoffmang9 what do you think?

Comment (mariano54) [reply src/plotting/create_plots.py:85]: This is used by the tests so it won't work.
I suggest changing it at the click level. Don't allow less than 32, unless a special flag is passed, like --override-k. What do you think @jespino ?

File: src/cmds/plots.py [hunk 2]
```diff
@@ -88,11 +90,11 @@ def plots_cmd(ctx: click.Context):
 def create_cmd(
     ctx: click.Context,
     size: int,
+    override_k: bool,
     num: int,
     buffer: int,
     num_threads: int,
     buckets: int,
-    stripe_size: int,
```
Comment (jespino) [inline src/cmds/plots.py:95]: This is fixing a bug that was there introduced by other commit.

**Review by hoffmang9 — APPROVED** [review]
looks good

**Pull request merged**